Userland/Libraries/LibWeb/WebSockets/WebSocket.cpp

```diff
@@ -107,18 +109,7 @@ WebSocket::ReadyState WebSocket::ready_state() const
 {
     if (!m_websocket)
         return WebSocket::ReadyState::Closed;
-    auto ready_state = const_cast<Protocol::WebSocket&>(*m_websocket).ready_state();
-    switch (ready_state) {
-    case Protocol::WebSocket::ReadyState::Connecting:
-        return WebSocket::ReadyState::Connecting;
-    case Protocol::WebSocket::ReadyState::Open:
-        return WebSocket::ReadyState::Open;
-    case Protocol::WebSocket::ReadyState::Closing:
-        return WebSocket::ReadyState::Closing;
-    case Protocol::WebSocket::ReadyState::Closed:
-        return WebSocket::ReadyState::Closed;
-    }
-    return WebSocket::ReadyState::Closed;
+    return const_cast<WebSocketClientSocket&>(*m_websocket).ready_state();
```

Comment:
Does ready_state() really need to be non-const? might be a change that needs followed all the way to the WebSocket library though, so this no-functional-change bit is probably fine.